lib/ansible/plugins/filter/core.py

```diff
@@ -486,6 +486,14 @@ def flatten(mylist, levels=None):
     return ret
 
 
+def dict_to_list_of_dict_key_value_elements(mydict):
+    ''' takes a dictionary and transforms it into a list of dictionaries,
+        with each having a 'key' and 'value' keys that correspond to the keys and values of the original '''
+    ret = []
```

Comment:
Should we ensure that `mydict` is actually a dict?

Could also "simplify" this with a list comprehension:

```
return [{'key': k, 'value': v} for k, v in mydict.items()]
```

Reply:
just copied the existing code, in any case that is 'shorten' not everyone thinks they are 'simpler'